app/controllers/content/vic_local_uploads_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Content
+  class VICLocalUploadsController < ApplicationController
+    skip_before_action :authenticate
+    # :nocov:
+    def find_file
+      path = Rails.root.join('public', params[:path])
```

Comment:
I know this is only active in local environments, but is it worth applying a little sanitization here and maybe a comment that describes why this is okay? I'd like to prevent the unlikely possibility that this gets copied and pasted somewhere...

Reply:
Yep! I'll add that in.